graphql/schema/types/config.graphql

```diff
@@ -17,6 +17,15 @@ type SpruceConfig {
   ui: UIConfig!
 }
 
+type AdminSettings {
+  announcements: Announcements
```

Comment:
APIAdminSettings doesn't have an announcements field and that's why there is generated rosolver code in `config_resolver.go`. You can avoid this by replacing announcements with Banner and BannerTheme instead since they already exist on [APIAdminSettings](https://github.com/evergreen-ci/evergreen/blob/64f77808e787c8522a51c647fdd996afd0de1a4b/rest/model/admin.go#L63-L64).